Epam/Java Basics_UpSkill Lab #1/Algorithmization/Decomposition_using_methods/src/com/androsor/decomp/HomeTask13.java

```diff
@@ -0,0 +1,46 @@
+package com.androsor.decomp;
+
+import java.util.Scanner;
+
+/**
+ * Two primes are called "twins" if they differ from each other by 2 (for example, 41 and 43). Find and print all pairs
+ * of "twins" from the segment [n, 2n], where n is a given natural number greater than 2.
+ * Use decomposition to solve the problem.
+ */
+
+public class HomeTask13 {
+
+    public static void main(String[] args) {
+
+        int n;
+
+        n = dataInput(" Введите число N = ");
+        System.out.printf(" На отрезке от %d до %d существуют следующие пары: \n", n, 2 * n);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.